internal/prometheus/prometheus.go

```diff
@@ -196,3 +200,31 @@ func (c *captureStatusResponseWriter) WriteHeader(statusCode int) {
 	c.ResponseWriter.WriteHeader(statusCode)
 	c.status = statusCode
 }
+
+func prettyByteSize(b uint64) string {
+	bf := float64(b)
+	for _, unit := range []string{"", "K", "M", "G", "T", "P", "E", "Z"} {
+		if math.Abs(bf) < 1000.0 {
+			return fmt.Sprintf("%3.1f%sB", bf, unit)
+		}
+		bf /= 1000.0
+	}
+	return fmt.Sprintf("%.1fYiB", bf)
+}
+
+func allocHandler(rsp http.ResponseWriter, req *http.Request) {
+	var m runtime.MemStats
+	runtime.ReadMemStats(&m)
+
+	total := m.HeapInuse + m.StackInuse + m.MCacheInuse + m.MSpanInuse
+
+	var alloc string
+	if req.URL.Query().Get("pretty") == "true" {
+		alloc = prettyByteSize(total)
+	} else {
+		alloc = fmt.Sprintf("%d", total)
+	}
+
+	rsp.WriteHeader(200)
+	_, _ = rsp.Write([]byte(fmt.Sprintf("%v\n", alloc)))
```

Comment:
Sorry for the last-minute nitpick:
```suggestion
	fmt.Fprintln(rsp, alloc)
```

Reply:
Neat. Fixed, thanks!